Localization/de-DE/Mods.PathOfTerraria.SkillPassives.hjson

```diff
@@ -205,3 +205,103 @@ Accelerant: {
 	// Name: Accelerant
 	// Tooltip: +{0}% increased sentinel life loss per second and +{0}% increased damage
 }
+
+BiggerBrood: {
+	// Name: Bigger Brood
+	// Tooltip: +1 egg per level
+}
+
+Gestation: {
+	// Name: Gestation
+	// Tooltip: 10% chance to spawn an additional egg when a locust kills an enemy
+}
+
+QuickerHatching: {
+	// Name: Quicker Hatching
+	// Tooltip: Eggs spawn 20% quicker per level
```

Comment:
For easier balancing later on it is a good practice to always use substitutions for numbers in locales.